pkgs/by-name/gr/gramps/package.nix

```diff
@@ -20,21 +20,16 @@
   ghostscript,
 }:
 
-let
-  inherit (python3Packages) buildPythonApplication pythonOlder;
-in
-buildPythonApplication rec {
-  version = "5.2.4";
+python3Packages.buildPythonApplication rec {
+  version = "6.0.0";
   pname = "gramps";
   pyproject = true;
 
-  disabled = pythonOlder "3.8";
```

Comment:
Why is this removed?
Wouldn't we want to keep this with `3.9` based on `setup.py`?

Reply:
I removed it as it is only really useful if it's in the range of python versions provided by nixpkgs (3.9-3.13)

Also, this is not a python package, but a python app: it will only evaluated with the current python3Packages set.